README.md

```diff
@@ -44,18 +44,20 @@ module.exports = {
     default: 'node index.js',
     lint: 'eslint .',
     test: {
-      // learn more about Jest here: https://kcd.im/egghead-jest
+      // learn more about Jest here: https://facebook.github.io/jest
       default: 'jest',
       watch: {
         script: 'jest --watch',
         description: 'run in the amazingly intelligent Jest watch mode'
       }
     },
     build: {
+      // learn more about Webpack here: https://webpack.js.org/
       default: 'webpack',
       prod: 'webpack -p',
     },
-    validate: 'nps --parallel lint,test,build',
+    // learn more about concurrently here: https://npm.im/concurrently
+    validate: 'concurrently "nps lint" "nps test" "nps build"',
```

Comment:
@tleunen the issue is that we now have to rely on how concurrently works, and it expects complete commands, not just the parameters like you would with `nps -p`